pkg/antctl/command_definition.go

```diff
@@ -135,15 +131,20 @@ func (e *resourceEndpoint) flags() []flagInfo {
 	if e.groupVersionResource == &v1beta2.NetworkPolicyVersionResource {
 		flags = append(flags, getSortByFlag())
 	}
+	if e.groupVersionResource == &v1beta2.AppliedToGroupVersionResource {
+		flags = append(flags, getSortByFlag())
+	}
+	if e.groupVersionResource == &v1beta2.AddressGroupVersionResource {
+		flags = append(flags, getSortByFlag())
+	}
```

Comment:
do you think it would make sense to add a new `supportsSorting` boolean field to the `resourceEndpoint` struct. This way, this could just be:
```go
if e.supportsSorting {
	flags = append(flags, getSortByFlag())
}
```

Reply:
I am not sure what you mean by overhead. This won't have an impact on performance.
And because the default value for boolean fields is `false` anyway, we won't even have to set the field explicitly for `resourceEndpoint` instances that don't support sorting.